### ISSUE TITLE
Handling cross-training accuracy labels

### ISSUE THREAD

**Issue by dsuplica** [AwhVogelLab/mvEEG#3]
Raising this issue to discuss how we want to handle labels for crosstraining. Currently, we assign each condition code a unique value based on group_dict which allows us to get accurate confidence scores, when the training dataset is a subset of the testing dataset (useful for hyperplane contrast within and across conditions). However, this messes up the accuracy score, because 50% of our output values are guaranteed to be wrong.

Options:
1. Keep as is, if you want accurate cross-decoding just make sure you assign group_dict properly and won't have the extra hyperplane contrast values. If you want hyperplane contrast you won't have reliable accuracy
2.  Calculate accuracy scores only for labels in the training set when the testing set contains extra labels
3. Calculate accuracy scores for subsets of the testing sets based on those which appear in the training set and which don't
